akka-http-core/src/main/scala/akka/http/scaladsl/model/HttpMessage.scala

```diff
@@ -417,7 +421,9 @@ final class HttpResponse(
 
   def mapEntity(f: ResponseEntity ⇒ ResponseEntity): HttpResponse = withEntity(f(entity))
 
-  /* Manual Case Class things, to easen bin-compat */
+  def transformEntityDataBytes[T](transformer: Graph[FlowShape[ByteString, ByteString], T]): HttpResponse = copy(entity = entity.transformDataBytes(Flow.fromGraph(transformer)))
```

Comment:
I see if that's also used in Java it would be problem with `Any` as Java doesn't respect or know about variance declarations.